src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts

```diff
@@ -250,6 +254,7 @@ export class ExtensionEditor extends BaseEditor {
 		this.navbar.onChange(this.onNavbarChange.bind(this, extension), this, this.transientDisposables);
 		this.navbar.push(NavbarSection.Readme, localize('details', "Details"));
 		this.navbar.push(NavbarSection.Contributions, localize('contributions', "Contributions"));
+		this.navbar.push(NavbarSection.Changelog, localize('changelog', "Changelog"));
```

Comment:
@joaomoreno 
I was wondering...giving we're fundamentally trying to get the extension changelog and this value is cached thanks to `Cache<string>`, I'd avoid to add a new method and try to get and store the value as I made here: e0f529fb8157366c32c590c0380d7b0bd693e3e7

Let me know your thoughts!